src/python/pants/backend/jvm/tasks/junit_run.py

```diff
@@ -90,10 +89,13 @@ def register_options(cls, register, register_jvm_tool):
                   'For example, 1/3 means run tests number 2, 5, 8, 11, ...')
     register('--suppress-output', action='store_true', default=True,
              help='Redirect test output to files in .pants.d/test/junit.')
-    # TODO(gmalmquist): Use the build root instead of the path of the first target by default.
     register('--cwd', default=_CWD_NOT_PRESENT, nargs='?',
-             help='Set the working directory. If no argument is passed, use the first target path. '
+             help='Set the working directory. If no argument is passed, use the build root. '
```

Comment:
If you change it to this behavior, you can simplify the _CWD_NOT_PRESENT argument handling which is confusing.  The build root is the CWD by default I think.